### PR TITLE
Upgrade to `cardano-api-8.32.0.0`

### PR DESCRIPTION
**Pull request by newhoggy**
# Changelog

```yaml
- description: |
    Upgrade to `cardano-api-8.32.0.0`
# uncomment types applicable to the change:
  type:
  # - feature        # introduces a new feature
  # - breaking       # the API has changed in a breaking way
  - compatible     # the API has changed but is non-breaking
  # - optimisation   # measurable performance improvements
  # - improvement    # QoL changes e.g. refactoring
  # - bugfix         # fixes a defect
  # - test           # fixes/modifies tests
  # - maintenance    # not directly related to the code
  # - release        # related to a new release preparation
  # - documentation  # change in code docs, haddocks...
```

# Context

Additional context for the PR goes here. If the PR fixes a particular issue please provide a [link](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword=) to the issue.

# How to trust this PR

Highlight important bits of the PR that will make the review faster. If there are commands the reviewer can run to observe the new behavior, describe them.

# Checklist

- [ ] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated. See [Running tests](https://github.com/input-output-hk/cardano-node-wiki/wiki/Running-tests) for more details
- [ ] Self-reviewed the diff

### CODE REVIEW

File: cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs [hunk 1]
```diff
@@ -413,205 +415,195 @@ runQueryKesPeriodInfoCmd
...
-                  (SystemStart $ protocolParamSystemStart gParams)
-                  (fromIntegral $ oCertExpiryKesPeriod * fromIntegral (protocolParamSlotsPerKESPeriod gParams))
-     in case time of
-          Left _ -> Nothing
-          Right t -> Just t
-
-   renderOpCertNodeAndOnDiskCounterInformation :: FilePath -> OpCertNodeAndOnDiskCounterInformation -> String
-   renderOpCertNodeAndOnDiskCounterInformation opCertFile opCertCounterInfo =
-     case opCertCounterInfo of
+  where
```
Comment (carlhammann) [inline cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs:418]: Why did you change the indentation? This makes the diff bigger than need be.

**Review by carlhammann — APPROVED** [review]

**Comment by jasagredo** [comment]
This PR has been absorbed by #463. May you close it @newhoggy ?
